scipy/stats/_stats_py.py

```diff
@@ -6996,19 +6996,18 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
     nobs2 : array_like
         The number(s) of observations of sample 2.
     equal_var : bool, optional
-        If True (default), perform a standard independent 2 sample test
-        that assumes equal population variances [1]_.
-        If False, perform Welch's t-test, which does not assume equal
-        population variance [2]_.
+        If True (default), perform a standard independent 2 sample test that
+        assumes equal population variances [1]_. If False, perform Welch's
+        t-test, which does not assume equal population variance [2]_.
     alternative : {'two-sided', 'less', 'greater'}, optional
-        Defines the alternative hypothesis.
-        The following options are available (default is 'two-sided'):
+        Defines the alternative hypothesis. The following options are available
+        (default is 'two-sided'):
 
         * 'two-sided': the means of the distributions are unequal.
-        * 'less': the mean of the first distribution is less than the
-          mean of the second distribution.
-        * 'greater': the mean of the first distribution is greater than the
-          mean of the second distribution.
+        * 'less': the mean of the first distribution is less than the mean of
+          the second distribution.
+        * 'greater': the mean of the first distribution is greater than the mean
+          of the second distribution.
```

Comment:
```suggestion
        * 'less': the mean of the first distribution is less than the
          mean of the second distribution.
        * 'greater': the mean of the first distribution is greater than the
          mean of the second distribution.
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.